mtags/src/main/scala/scala/meta/internal/pc/HoverProvider.scala

```diff
@@ -1,17 +1,19 @@
 package scala.meta.internal.pc
 
-import org.eclipse.lsp4j.jsonrpc.messages.{Either => JEither}
 import org.eclipse.lsp4j.Hover
-import org.eclipse.lsp4j.MarkedString
+import org.eclipse.lsp4j.MarkupContent
+import org.eclipse.lsp4j.MarkupKind
 import scala.meta.internal.mtags.MtagsEnrichments._
 import scala.meta.pc.OffsetParams
 import scala.reflect.internal.{Flags => gf}
-import scala.meta.internal.jdk.CollectionConverters._
 import scala.util.control.NonFatal
 
 class HoverProvider(val compiler: MetalsGlobal, params: OffsetParams) {
   import compiler._
 
+  private val scalaMarkdownTics = "```scala"
+  private val endTics = "```"
```

Comment:
I have this in here twice. Also in the `DecorationWorksheetPublisher`. Should I just leave as is, or do you think I should combine them and put them somewhere? Mainly I wasn't sure where to stick them.

Reply:
maybe we could have method like `makeScalaMarkupContent(body)` that would handle all boilerplate(making markupContent with kind Markdown and wrapping body in scala tag). We could use that in both places.

Reply:
Sounds good. I realized we had a `HoverMarkup` object that I could reuse, and just added another apply to it.